hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/snapshot/SnapshotDiffManager.java

```diff
@@ -367,6 +367,17 @@ private Set<String> getSSTFileListForSnapshot(OmSnapshot snapshot,
         .getPath(), tablesToLookUp);
   }
 
+  /**
+   * Gets the report key for a particular index of snapshot diff job.
+   * @param jobId Snapshot diff jobId
+   * @param index
+   * @return report Key of the snapshot diff job
+   */
+
+  static String getReportKeyForIndex(String jobId, long index) {
+    return String.format("%s%s%020d", jobId, DELIMITER, index);
```

Comment:
My hunch is using a format String might be slower than using `StringUtils.leftPad()` on `index` (or even hand-rolling) because of the overhead of parsing the format String itself.

Not a major concern for now. But we can swap this out later.

Reply:
done.

Reply:
Can you check if this is what you were looking for